src/Nest/CommonAbstractions/Extensions/ExpressionExtensions.cs

```diff
@@ -31,7 +31,7 @@ public SuffixExpressionVisitor(string suffix)
 				this.suffix = suffix;
 			}
 
-			protected override Expression VisitMember(MemberExpression node)
+			public override Expression Visit(Expression node)
```

Comment:
@gmarz, well I don't think it makes any difference. The public `Visit(Expression node)` method is called anyway and by default "dispatches the expression to one of the more specialized visit methods in this class". Thus the protected `VisitMember(MemberExpression node)` is called by the public `Visit(Expression node)` method. But since we want to append the suffix to the whole expression, not just to the specialized parts of it, I can't see any other way than directly override the public method. Or did I miss something?